public/index.html

```diff
@@ -4,20 +4,19 @@
     <meta charset="utf-8">
     <title>PG workshop - get users from the database</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.7/css/materialize.min.css">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
   </head>
   <body>
     <nav>
       <div class="nav-wrapper">
         <a href="#" class="brand-logo">Users</a>
       </div>
     </nav>
-    <!-- <form class="form" action="/create-user" method="post">
+    <form class="form" action="/create-user" method="post">
```

Comment:
Any reason you don't have this commented out any more? You've still got the instruction above, that says "un-comment out the form" https://github.com/foundersandcoders/pg-workshop/pull/22/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38

Having this from present the start means that the first time someone sees this in the browser, they'll probably try to add a new user & not understand why it doesn't work